### ISSUE TITLE
polyfill returns invalid sourceMappingUrl and kills app in Safari 9.1

### ISSUE THREAD

**Issue by bsangster-classy** [polyfillpolyfill/polyfill-service#1244]
To reproduce:
`$ curl 'https://cdn.polyfill.io/v2/polyfill.min.js?features=Intl.~locale.en' -H 'Accept-Language: en-US,en;q=0.8' -H 'Upgrade-Insecure-Requests: 1' -H 'User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11) AppleWebKit/602.1.13 (KHTML, like Gecko) Version/9.1.8 Safari/602.1.13' -H 'Accept: text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8' -H 'Cache-Control: max-age=0' -H 'Connection: keep-alive' | grep "source"`

Result:
`//# sourceMappingURL=Intl.min.js.map`

Requesting that file:
`$ curl 'https://cdn.polyfill.io/v2/Intl.min.js.map' -H 'Accept-Language: en-US,en;q=0.8' -H 'Upgrade-Insecure-Requests: 1' -H 'User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11) AppleWebKit/602.1.13 (KHTML, like Gecko) Version/9.1.8 Safari/602.1.13' -H 'Accept: text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8' -H 'Cache-Control: max-age=0' -H 'Connection: keep-alive'`

Results in:
`Cannot GET /v2/Intl.min.js.map`

Note that while other browsers seem to handle this okay, Safari 9.X is dying and showing us white screens (apparently) because it fails to properly load the original JS file.

**Comment by davidensinger** [comment]
Hi, I have this exact issue with `//# sourceMappingURL=Intl.min.js.map` failing on Safari 9.x and it unfortunately appears to persist, despite #1283.

To reproduce: https://cdn.polyfill.io/v2/polyfill.min.js?features=Intl.~locale.en&flags=always

If you search within that file you’ll find the offending source map comment which unfortunately gets evaluated by Safari 9.x. Since the source map isn’t found it kills any subsequent JS.

Please let me know if you need any more information from me and thank you for your efforts in maintaining such a great service!

**Comment by bsangster-classy** [comment]
@JakeChampion I have also confirmed that this is not fixed on either prod or the current QA build:

`curl 'https://qa.polyfill.io/v2/polyfill.min.js?features=Intl.~locale.en' -H 'Accept-Language: en-US,en;q=0.8' -H 'Upgrade-Insecure-Requests: 1' -H 'User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11) AppleWebKit/602.1.13 (KHTML, like Gecko) Version/9.1.8 Safari/602.1.13' -H 'Accept: text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8' -H 'Cache-Control: max-age=0' -H 'Connection: keep-alive' | grep "source"`

I hacked around it a few months ago on our end, but figured you might want to know.

I peeked at your unit test briefly, and suspect that the reason it isn't failing is because you aren't specifying the Safari 9 UA and/or aren't asking for Intl. Did you check that the test failed before your fix was added?
